PWR/PWR_button/Core/Src/main.c

```diff
@@ -0,0 +1,182 @@
+#include "stm32l053xx.h"
+
+#define BUTTON_DEBOUNCE	50
+
+uint8_t flag_EXTI 		= 0,
+		user_bt_count	= 0;	//counter to debounce the button
+
+void MCU_Init(void);
+
+void EXTI4_15_IRQHandler (void)
+{
+	if(EXTI->PR & EXTI_PR_PIF10)	//if there's EXTI interrupt
+	{
+		flag_EXTI = 1;
+		EXTI->PR |= EXTI_PR_PIF10;	//clears the EXTI flag
+	}
+}
+
+int main(void)
+{
+	MCU_Init();
+
+	while(1)
+	{
+		if(flag_EXTI)
```

Comment:
It is a good practice to always zero flags after checking them. It also reinforce that the flag is really only being used as a flag.